confluence/confluence.go

```diff
@@ -0,0 +1,118 @@
+package confluence
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/JackTimothy/pubdoc/configuration"
+	"github.com/JackTimothy/pubdoc/parser"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+func FormatBase(base string) (string, error) {
+	ext := path.Ext(base)
+	fmtBase, ok := strings.CutSuffix(base, ext)
+	if !ok {
+		return "", fmt.Errorf("Could not find extension suffix")
```

Comment:
Go staticcheck linter complaining about the capital "C" since error strings aren't supposed to be capitalized